app/views/papers/index.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= render 'courses/pickCourse'%>
+<div>
+
+</p>
+
+	<h2 ><%= @course.code %> <%= @semester.start_year %>/<%= @semester.end_year %> Sem <%= @semester.number %> Papers </h2>
```

Comment:
Better to define this in the model. Can take a look at my `Semester.formatted` method in the model